common/src/main/java/com/wynntils/features/user/players/PlayerGhostTransparencyFeature.java

```diff
@@ -30,4 +34,13 @@ public void onTranslucentCheck(LivingEntityRenderTranslucentCheckEvent e) {
             e.setTranslucence(playerGhostTranslucenceLevel);
         }
     }
+
+    @SubscribeEvent
+    public void onGhostArmorRender(PlayerGhostArmorRenderEvent e) {
+        if (!WynnUtils.onWorld()) return;
+
+        if (WynnPlayerUtils.isPlayerGhost((Player) e.getEntity())) {
```

Comment:
This cast does not look good. When reading this code, the question "how do you know this is a Player" pops into your mind, and the gut reaction is to add an instance test first. But I continued through the code, and found the answer in a very different place. I made a comment there about how this needs to be fixed. And when that is done, this cast can be removed.